nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/authorizers.xml

```diff
@@ -198,12 +198,22 @@
             NOTE: Any identity mapping rules specified in nifi.properties are not applied in this implementation. This behavior
             would need to be applied by the base implementation.
     -->
```

Comment:
The block comment above the commented-out `<userGroupProvider>` declaration should explain the Shell User Group Provider and its elements. The block comment for `CompositeConfigurableUserGroupProvider` should be moved below the commented-out Shell UGP.

Reply:
Good catch, thank you.  Commented.